docs/diagrams/src/DataStoring/studentIdentification.puml

```diff
@@ -0,0 +1,62 @@
+@startuml Student Identification
+footbox off
+
+participant Class as class
+participant findStudentByName as fsbname
+database "masterStudentList:Student[ ]" as msl
+participant findStudentByNumber as fsbnumber
+participant User as user
```

Comment:
Good choice of placing the User at the far right most of the sequence diagram to increase readability.